module/VuFind/tests/unit-tests/src/VuFindTest/View/Helper/Root/IconTest.php

```diff
@@ -130,9 +131,15 @@ protected function getIconHelper(
         $icon = new Icon(
             $config ?? $this->getDefaultTestConfig(),
             $cache ?? new BlackHole(),
-            new EscapeHtmlAttr(),
+            new EscapeHtmlAttr(new \VuFind\Escaper\Escaper(false)),
```

Comment:
Would it be better to assign `$escapeHtmlAttr = new \VuFind\Escaper\Escaper(false);` up above, so it can be used here and also as `compact('escapeHtmlAttr')` down below to avoid redundant construction and shave a couple of lines off the code?

Also, would it make sense to give the Escape constructor parameter a default value to reduce verbosity in all these tests, or do you think it's important to make this an explicit decision?